cli/tests/package-search.bats

```diff
@@ -81,15 +81,15 @@ setup_file() {
 
 @test "'FLOX_FEATURES_SEARCH_STRATEGY=match flox search' expected number of results: 'hello'" {
   FLOX_FEATURES_SEARCH_STRATEGY=match run --separate-stderr "$FLOX_BIN" search hello --all
-  assert_equal "${#lines[@]}" 11
```

Comment:
Are these tests really sensitive to the nixpkgs we build flox with?
I thought of those to be basically _some independent_ rev.

would be better to isolate them a bit more so we can do future updates without changing lots of tests that depend on specific _locked_ results...

Reply:
Yes, they are sensitive to the nixpkgs version. One possibility is keep testing on the same nixpkgs as before. That might be enough, see revision f44164dfabbd207cb27e39e82bb8f6c6dd8cb631 (two commits back). This passes all tests. @mkenigs I forgot to mention this, it could be a reasonable stepping stone before trying to change the tests.